aws-xray/src/main/java/io/opentelemetry/contrib/awsxray/AwsXrayRemoteSampler.java

```diff
@@ -106,11 +113,20 @@ private void getAndUpdateSampler() {
     } catch (Throwable t) {
       logger.log(Level.FINE, "Failed to update sampler", t);
     }
+    scheduleSamplerUpdate();
```

Comment:
I didn't follow the current pattern of xray sdk which schedules a fixed rate with jitter

https://github.com/aws/aws-xray-sdk-java/blame/5143932f0da7d921c07fc8149d14c3f767cb09ee/aws-xray-recorder-sdk-core/src/main/java/com/amazonaws/xray/strategy/sampling/pollers/RulePoller.java#L80

It's unfair scheduling, as some processes will forever have a lower polling rate and some will forever have a higher one. The intent should always be to make sure over the long term the semantics are the same for any process, so we need to apply the jitter per-schedule for it to be fair.

Of course, the downside is at any point of time the randoms can align to a thundering herd. I don't think it can be avoided though and is why backends need (and I figure have) rate limits anyways.

Reply:
Yep, I prefer your approach here.